backend-project/small_eod/letters/views.py

```diff
@@ -52,7 +52,7 @@ class FileViewSet(
     serializer_class = FileSerializer
 
     def get_queryset(self):
-        return self.model.objects.filter(letter__pk=self.kwargs["letter_pk"]).all()
+        return File.objects.filter(letter__pk=self.kwargs["letter_pk"]).all()
```

Comment:
Czy to jest celowa zmiana? Mam wrażenie, że nie, ale jeżeli tak to muszę się temu bliżej przyjrzeć, co za magia się dzieje, że `self.model != File`.

Reply:
Chodziło mi o wskazanie modelu bardziej explicite. Mając `self.model` trzeba patrzeć jaki jest model, natomiast przy takim zapisie jest to wyraźnie wskazane.

Reply:
Już nazwa klasy wskazuje na typ modelu, a w takim ujęciu więcej pracy i trudniej dostrzec wzorce, aby wyabstrahować. Przestają także działać Mixiny, a będziemy mieli je np. dotyczące uprawnień. Ja proponuje jeszcze prościej to zapisać korzystając z tego, że `GenericViewSet` dziedziczy z `GenericAPIView`:
```suggestion
        return super().get_queryset().filter(letter__pk=self.kwargs["letter_pk"])
```